queries.sql

```diff
@@ -8,3 +8,55 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered > 0;
 SELECT * from animals WHERE name = 'Gabumon';
 SELECT * from animals WHERE 10.4 <= weight_kg <= 17.3;
+
+/*ADD a column and 2nd. pt. Queries*/
+
+ALTER TABLE animals ADD species varchar(100);
+
+BEGIN;
+
+UPDATE animals SET species = 'Digimon' WHERE name = 'Agumon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Gabumon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Devimon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Plantmon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Angemon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Boarmon';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Pikachu';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Charmander';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Squirtle';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Blossom';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Ditto';
+
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'Pokemon' WHERE species = NULL;
+
+COMMIT;
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
```

Comment:
- [ ] In this transaction, after rolling back, you are required to confirm that the transaction was not committed. To do this, you can write this line of code `SELECT COUNT(*) FROM ANIMALS;`. Kindly add this line of code. 👍